packages/components/organisms/f-checkout/src/components/Error.vue

```diff
@@ -22,23 +21,78 @@
             data-test-id="checkout-error-page-description">
             {{ $t(`errorMessages.${errorFormType}.description`) }}
         </p>
+
+        <f-button
+            :class="$style['c-checkout-error-button']"
+            button-size="large"
+            button-type="primary"
+            is-full-width
+            data-test-id="error-page-redirect-to-menu-button"
+            @click.native="redirectToMenu">
+            {{ $t(`errorMessages.${errorFormType}.buttonText`) }}
+        </f-button>
     </card>
 </template>
 
 <script>
+import { mapState } from 'vuex';
 import Card from '@justeat/f-card';
+import FButton from '@justeat/f-button';
 import SadBagIconDecorator from '../assets/images/jet-sad-bag.svg';
+import '@justeat/f-button/dist/f-button.css';
 import '@justeat/f-card/dist/f-card.css';
+import {
+    VUEX_CHECKOUT_MODULE,
+    CHEKOUT_ERROR_FORM_TYPE
+} from '../constants';
+import loggerMixin from '../mixins/logger.mixin';
 
 export default {
     components: {
         Card,
+        FButton,
         SadBagIconDecorator
     },
+    mixins: [
+        loggerMixin
+    ],
     props: {
         errorFormType: {
             type: String,
             required: true
+        },
+        restaurantMenuPageUrl: {
```

Comment:
I'm always a bit wary of naming things related to pages, as they can change down the line.

Would a name like `redirectUrl` be appropriate (as it looks like that is its actual function?)